library/ssl_tls13_keys.c

```diff
@@ -1019,14 +1019,14 @@ int mbedtls_ssl_tls13_populate_transform(
 
```

Comment:
Preferably we'd include a [changelog entry](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md), but we don't consistently write one when we're just silencing a compiler warning.